webrender/src/frame_builder.rs

```diff
@@ -129,6 +129,9 @@ pub struct FrameBuilder {
     /// parent new scroll nodes.
     reference_frame_stack: Vec<ClipId>,
 
+    parent_scroll_frame_id: Option<ClipId>,
+    pending_scroll_frame_stack: Vec<(ClipId, ClipId, LayerRect)>,
+
```

Comment:
I doubt think this _needs_ to be a Vec, I don't think there will be more than one pending scroll frame that hasn't drawn scrollbars at a time.